analyzers/DNSDB/dnsdb_query.py

```diff
@@ -14,39 +14,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Python3 compatability by: https://github.com/guyddr/dnsdb-query
```

Comment:
This is a Python3 version of this vendor script we imported from elsewhere. Link listed here.